### PR TITLE
Updated formatting for consistency.

### PR DESCRIPTION
**Pull request by JoinedSenses**
This PR is perhaps overkill, however this is something I've wanted to get to for a few months. 95% of the changes are moving from tabs to spaces for aligning text (not initial indents, which have remained as tabs)

Other misc changes:
-Added missing @deprecated comments
-Modified braces on stock functions for consistency
-Removed empty spaces at the end of lines,
-Removed commas following last enum member.
-A bunch of other minor junk that I can't remember

Most tab to space conversions were done through regexes:
```
// used for finding tabs
"((?<!^)\t)|(?<= )\t"

// 1 through 13 while decreasing no. of spaces by 1 each interation.
"param ([\w\.]{1})\t+"
"param \1             "

"@return[ \t]+"
"@return              "

"@error[ \t]+"
"@error               "

"\*[ \t]{3,}"
"*                      "

"[ \t]+$"
""
```

Everything else was done manually with the use of beer and music.

### CODE REVIEW

File: plugins/include/sdktools_sound.inc [hunk 1]
```diff
@@ -466,31 +467,29 @@ stock void EmitSoundToAll(const char[] sample,
...
 	}
-	
-	if (!total)
+
+	if (total)
 	{
-		return;
+		EmitSound(clients, total, sample, entity, channel,
+			level, flags, volume, pitch, speakerentity,
+			origin, dir, updatePos, soundtime);
```
Comment (JoinedSenses) [inline plugins/include/sdktools_sound.inc]: Moved code inside of codeblock instead of returning.

File: plugins/include/sdktools_sound.inc [hunk 2]
```diff
@@ -606,16 +603,14 @@ stock bool EmitAmbientGameSound(const char[] gameSound,
...
 	{
 		EmitAmbientSound(sample, pos, entity, level, flags, volume, pitch, delay);
 		return true;
 	}
-	else
-	{
-		return false;
-	}
+
+	return false;
```
Comment (JoinedSenses) [inline plugins/include/sdktools_sound.inc]: Removed unecessary else statements.

File: plugins/include/events.inc [hunk 3]
```diff
@@ -339,4 +340,4 @@ native void GetEventName(Handle event, char[] name, int maxlength);
  * @param dontBroadcast True to disable broadcasting, false otherwise.
  * @error               Invalid Handle.
  */
-native void SetEventBroadcast(Handle event, bool dontBroadcast);
\ No newline at end of file
+native void SetEventBroadcast(Handle event, bool dontBroadcast);
```
Comment (JoinedSenses) [inline plugins/include/events.inc]: Added EOF

File: plugins/include/sdkhooks.inc [hunk 4]
```diff
@@ -35,68 +35,70 @@
...
-#define DMG_REMOVENORAGDOLL				(1 << 22)	/**< with this bit OR'd in, no ragdoll will be created, and the target will be quietly removed.
-																								 use this to kill an entity that you've already got a server-side ragdoll for */
-#define DMG_PHYSGUN								(1 << 23)	/**< Hit by manipulator. Usually doesn't do any damage. */
-#define DMG_PLASMA								(1 << 24)	/**< Shot by Cremator */
-#define DMG_AIRBOAT								(1 << 25)	/**< Hit by the airboat's gun */
-#define DMG_DISSOLVE							(1 << 26)	/**< Dissolving! */
-#define DMG_BLAST_SURFACE					(1 << 27)	/**< A blast on the surface of water that cannot harm things underwater */
-#define DMG_DIRECT								(1 << 28)
-#define DMG_BUCKSHOT							(1 << 29)	/**< not quite a bullet. Little, rounder, different. */
+#define DMG_GENERIC                  0          /**< generic damage was done */
```
Comment (JoinedSenses) [inline plugins/include/sdkhooks.inc]: Now more pretty.

File: plugins/include/sdktools_variant_t.inc [hunk 5]
```diff
@@ -31,63 +31,63 @@
...
  * Sets an entity in the global variant object.
  *
- * @param entity	Entity index.
- * @error		Invalid entity index.
+ * @param entity        Entity index.
+ * @error               Invalid entity index.
  */
-native void SetVariantEntity(int entity);
\ No newline at end of file
+native void SetVariantEntity(int entity);
```
Comment (JoinedSenses) [inline plugins/include/sdktools_variant_t.inc]: Added EOF

File: plugins/include/basecomm.inc [hunk 6]
```diff
@@ -38,50 +38,50 @@
...
  
  /**
  * Called when a client is gagged or ungagged
  * 
- * @param	client		Client index
- * @param	gagState	True if client was gaged, false otherwise
+ * @param   client      Client index
+ * @param   gagState    True if client was gaged, false otherwise
  */
- forward void BaseComm_OnClientGag(int client, bool gagState);
```
Comment (JoinedSenses) [inline plugins/include/basecomm.inc]: Removed some spaces at start of code.

File: plugins/include/adt_array.inc [hunk 7]
```diff
@@ -208,20 +210,16 @@ methodmap ArrayList < Handle {
...
-	/**
-	 * Sort an ADT Array. Specify the type as Integer, Float, or String.
-	 *
-	 * @param order			Sort order to use, same as other sorts.
-	 * @param type			Data type stored in the ADT Array
-	 */
+	// Sort an ADT Array. Specify the type as Integer, Float, or String.
+	//
+	// @param order         Sort order to use, same as other sorts.
+	// @param type          Data type stored in the ADT Array
```
Comment (JoinedSenses) [inline plugins/include/adt_array.inc]: Modified comment style. Using `//` for methodmaps and `/**/` for natives/forwards

File: plugins/include/commandline.inc [hunk 8]
```diff
@@ -31,56 +31,56 @@
  */
 
 #if defined _commandline_included_
-  #endinput
+ #endinput
```
Comment (JoinedSenses) [inline plugins/include/commandline.inc]: Modified preprocessor directives to indent with spaces for consistency.

File: plugins/include/adt_array.inc (hunk 7 above)
Comment (Headline) [reply plugins/include/adt_array.inc]: good catch

**Pull request merged**